monai/handlers/tensorboard_handlers.py

```diff
@@ -87,12 +87,14 @@ class TensorBoardStatsHandler(TensorBoardHandler):
 
     """
 
+    @deprecated_arg("epoch_interval", since="1.1", removed="1.3")
+    @deprecated_arg("iteration_interval", since="1.1", removed="1.3")
```

Comment:
Could you please help add some msg_suffix  message here to help user understand?
CC @wyli 

Thanks.

Reply:
I added `msg_suffix` as we discussed in the slack but it gives a redundant message like:
```
monai.handlers.tensorboard_handlers TensorBoardStatsHandler.__init__:iteration_interval: Argument `iteration_interval` has been deprecated since version 1.1. It will be removed in vers
ion 1.3. ``iteration_interval`` has been deprecated and will be removed soon. 
```
So, that's why I omitted it. Right now we can't suggest any replacement for the arg.

Reply:
thanks both, that looks like a bug in the ci/cd for deprecated_arg, I'll fix it and merge this PR soon.

Reply:
Maybe related: https://github.com/python-versioneer/python-versioneer/issues/206

Reply:
yes, the version is not well defined in this case, and this could happen in a few use cases as well. I think `deprecated_arg` should assume the arg is deprecated but not removed when monai version is unknown..